lib/renderer.js

```diff
@@ -745,8 +745,8 @@
      *
      * @private
      * @param {Component} component
-     * @return {Promise.<{inputRect: Bounds, outputRect: Bounds, expectedWidth: number,
-     *      expectedHeight: number, getPadding: function (number, number): number }>}
+     * @return {Promise.<{inputRect: Bounds, outputRect: Bounds, 
```

Comment:
Since we have this documented here, might as well indicate that inputRect and outputRect may be undefined, right?  `Bounds=`